dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/Body.java

```diff
@@ -0,0 +1,23 @@
+package com.dotmarketing.portlets.templates.design.bean;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * It's a {@link com.dotmarketing.portlets.templates.model.Template}'s Body
+ */
+public class Body {
+    private List<TemplateLayoutRow> rows;
+
+
+    @JsonCreator
+    public Body(@JsonProperty("rows") List<TemplateLayoutRow> rows) {
+        this.rows = rows;
+    }
+
+    public List<TemplateLayoutRow> getRows() {
+        return rows;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "rows". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)